doc/blog-post/2017-07-xx-1.3.0-announcement.md

```diff
@@ -0,0 +1,160 @@
+
+## Dynamically Augment Date Skeletons
+
+> This change was contributed by Marat Dyatko @vectart and Artur Eshenbrener @Strate in #462 and #604.
+
+The style used to display a date format often vary depending on the application. CLDR offers data for certain presets like short, medium, long, and full vs. date, time, and datetime. It also offers data for a lot of different [date fields](http://www.unicode.org/reports/tr35/tr35-dates.html#Date_Field_Symbol_Table) and their combinations, which are used by globalize to sintetize an open-ended list of custom formats (called skeletons). But, what's interesting is that it would be prohibitively large if CLDR provided data for every single possible combination. So, there's an algorithm specified by UTS#35 to deduce missing data from the requested format.
+
+For example, there are four different presets to format a date and they would generate `"7/1/17"` (short), `"Jul 1, 2017"` (medium), `"July 1, 2017"` (long), and `"Saturday, July 1, 2017"` (full). Although, let's pretend we want something different such as `"Jul 1"`. For that format we could use `{skeleton: "MMMd"}`. Internally, globalize finds a direct match (in CLDR) for that skeleton. Let's now pretend we want `"July 1"`. For that format we could use `{skeleton: "MMMMd"}`. Internally, globalize doesn't find a direct match (in CLDR) for this skeleton and it needs to use the data for `MMMd`, which in the case of English maps to `"MMM d"`, and then replace `MMM` with `MMMM` dynamically generating `"MMMM d"`. If we wanted `"07/01"` instead, we could use `{skeleton: "MMdd"}`. Internally, globalize doesn't find a direct match in CLDR for this skeleton and it needs to use the data for `Md`, which in the case of English maps to `"M/d"`, and then replace `M` wtih `MM` and `d` with `dd` dynamically generating `"MM/dd"`.
+
+To make a long story short, the algorithm in globalize 1.3.0 has been significantly improved and it allows using virtually any skeletons except for some special cases.
+
+```js
+let skeleton = "GyMMMMEEEEdhms";
+Globalize("en").formatDate(new Date(), {skeleton});
+// > 'Saturday, July 1, 2017 AD at 4:58:27 PM'
+Globalize("pt").formatDate(new Date(), {skeleton});
+// > 'sábado, 1 de julho de 2017 d.C. 5:01:20 PM'
+Globalize("de").formatDate(new Date(), {skeleton});
+// > 'Samstag, 1. Juli 2017 n. Chr. um 5:01:33 nachm.'
+Globalize("zh").formatDate(new Date(), {skeleton});
+// > '公元2017年七月月1日星期六 下午5:01:35'
+Globalize("ko").formatDate(new Date(), {skeleton});
+// > 'AD 2017년 7월 1일 토요일 오후 5:01:38'
+Globalize("ar").formatDate(new Date(), {skeleton});
+// > 'السبت، ١ يوليو، ٢٠١٧ م ٥:٠١:٤٠ م'
+Globalize("ar-MA").formatDate(new Date(), {skeleton});
+// > 'السبت، 1 يوليوز، 2017 م 5:04:29 م'
+Globalize("it").formatDate(new Date(), {skeleton});
+// > 'sabato 1 luglio 2017 d.C. 5:01:52 PM'
+```
+
+Read our [getting started](https://github.com/globalizejs/globalize/#getting-started) and play with it yourself.
+
+## Other Enhancements and Bug Fixes
```

Comment:
Not sure in what format this is supposed to be published, but currently all the PR and issue references aren't links. Above there's enough content to ignore that, but the following is lacking context to be useful without links.